### PR TITLE
Feature/refactoring js files

### PR DESCRIPTION
**Pull request by roht0515**
refactoring the use of scripts for each view

### CODE REVIEW

**Review by Laura4lilavati — COMMENTED** [review]
Can you add indentations where needed in *.hmtl files?

**Review by MirkoFer-622 — COMMENTED** [review]
I think that, as all project's classes are named using Camel Case starting with uppercase letters, you should change names using this agreement

**Comment by MirkoFer-622** [comment]
I also consider that you have to name files using class names (i.e. ExtractText.html instead doc.html). The same for .js files (i.e. you should change img.js for Image.js)

File: src/main/resources/templates/metadata/metadata.html [hunk 1]
```diff
@@ -94,5 +94,6 @@ <h6 class="m-3 align-bottom">Metadata</h6>
 </div>
 <footer th:replace="main/template :: footer">
 </footer>
+<script th:src="@{/js/metadata/metadata.js}"></script>
```
Comment (maryangela) [inline src/main/resources/templates/metadata/metadata.html]: please add identation

**Review by Laura4lilavati — APPROVED** [review]

**Review by maryangela — APPROVED** [review]

**Pull request merged**